touchforms/formplayer/templates/touchforms/touchscreen.html

```diff
@@ -18,8 +18,8 @@
     <script language="javascript" type="text/javascript" src="{{STATIC_URL}}formplayer/script/deprecate/entrycontrols.js?version={{app_version}}"></script>
     <script language="javascript" type="text/javascript;version=1.7" src="{{STATIC_URL}}formplayer/script/decprecate/workflow.js?version={{app_version}}"></script>
    
-    <script language="javascript" type="text/javascript" src="{{STATIC_URL}}formplayer/script/deprecate/jquery-1.4.2.min.js"></script>
-    <script language="javascript" type="text/javascript" src="{{STATIC_URL}}formplayer/script/vendor/shortcut.js"></script>
+    <script language="javascript" type="text/javascript" src="{{STATIC_URL}}formplayer/script/jquery-1.4.2.min.js"></script>
+    <script language="javascript" type="text/javascript" src="{{STATIC_URL}}formplayer/script/shortcut.js"></script>
```

Comment:
this change will cause 404s (and likely other errors) if touchscreen.html is loaded